packages/neuron-ui/src/components/Addresses/index.tsx

```diff
@@ -181,18 +186,44 @@ const Addresses = ({
         enableShimmer={isLoading}
         checkboxVisibility={CheckboxVisibility.hidden}
         columns={addressColumns.map(col => ({ ...col, name: t(col.name) }))}
-        items={addresses}
+        items={addresses.map(addr => ({
+          ...addr,
+          address: showMainnetAddress
+            ? ckbCore.utils.bech32Address(addr.identifier, {
+                prefix: ckbCore.utils.AddressPrefix.Mainnet,
+                type: ckbCore.utils.AddressType.HashIdx,
+                codeHashIndex: '0x00',
+              }) || ''
+            : addr.address,
```

Comment:
Is this assuming current network is always `testnet`? It should check current network. (I know `network.type` is something not implemented, but want to know that this would get updated and delegate to that property).

Reply:
Same for `showMainnetAddress`'s initial state.

Reply:
I've added a condition to determine whether the toggle and caution should be displayed in https://github.com/nervosnetwork/neuron/pull/939/commits/5be03351d1c28b90c88b887024aa0a8a4c2ad156